plenum/common/messages/fields.py

```diff
@@ -133,6 +134,25 @@ def _specific_validation(self, val):
             return '{} is longer than {} symbols'.format(val, self._max_length)
 
 
+class DatetimeStringField(FieldBase):
+    _base_types = (str,)
+    _exception_values = []
+
+    def __init__(self, exception_values: []=None, **kwargs):
```

Comment:
Could you please rename exception_values to exceptional_values
The same with self._exception_values

Reply:
done